packages/3dt/build

```diff
@@ -22,19 +22,23 @@ mkdir -p $(dirname $slave_public_service)
 cat <<EOF | tee $slave_service $slave_public_service
 [Unit]
 Description=DC/OS Diagnostics (3DT) Agent: exposes component health
+
 [Service]
 EnvironmentFile=/opt/mesosphere/environment
 Restart=always
 StartLimitInterval=0
 RestartSec=5
 LimitNOFILE=16384
 User=dcos_3dt
+# Allow r/w access to the socket file
```

Comment:
There was a merge conflict with the refactor of 3dt happening subsequently here (https://github.com/dcos/dcos/pull/1287/commits/54154cfee759bf44a9876da6823954bf209ea308) 

It has been resolved in the PR (https://github.com/dcos/dcos/pull/1336) please look out for that @gpaul  and @darkonie .